python/cudf/cudf/core/series.py

```diff
@@ -462,15 +471,10 @@ def __setitem__(self, key, value):
 
         self._column[key] = value
 
-    def take(self, indices, ignore_index=False):
+    def take(self, indices):
         """Return Series by taking values from the corresponding *indices*.
         """
-        result = self[indices]
-        if ignore_index:
-            index = RangeIndex(len(result))
-            return result._copy_construct(index=index)
-        else:
-            return result
+        return self[indices]
```

Comment:
This looks like it will do a `.loc` by default instead of an `.iloc`. Existing bug from the looks of it, but wanted to note.